kuberay/manifests/ray-cluster.tpu-v4-multihost.yaml

```diff
@@ -0,0 +1,144 @@
+# This template contains a Kuberay cluster using a 2x2x2 TPU v4 PodSlice.
+# To get access to TPU resources, please follow instructions in this link:
+# https://cloud.google.com/kubernetes-engine/docs/how-to/tpus
+apiVersion: ray.io/v1
+kind: RayCluster
+metadata:
+  name: example-cluster-kuberay
+spec:
+  headGroupSpec:
+    rayStartParams:
+      {}
+    template:
+      spec:
+        imagePullSecrets:
+          []
+        serviceAccountName: ray-ksa
+        containers:
+          - volumeMounts:
+            - name: gcs-fuse-checkpoint
+              mountPath: /llama
+              readOnly: true
+            - mountPath: /tmp/ray
+              name: ray-logs
+            name: ray-head
+            image: gcr.io/tpu-vm-gke-testing/ricliu-jetstream:20240709
```

Comment:
This image is not publically available and references an internal project, can we host it on a public registry or provide the Dockerfile so users can build it themselves?

Reply:
cc @ryanaoleary

Reply:
Ah I see the Dockerfile now. I suggest not referencing the private image though if possible.